conda-forge.yml

```diff
@@ -4,5 +4,3 @@ travis:
 appveyor:
   secure:
     BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
-provider:
-  win: azure
```

Comment:
The failure on Python 2.7 is related to vc9 and not AppVeyor vs Azure. Let's keeo Azure here b/c it is faster and we are slowly moving away from AppVeyor.

Reply:
Ok, got it. From the documentation page it looked like Azure can't do Python 2.7 builds for Windows at all, but i see there's an error in AppVeyor too. I'll close this without merging, and try again when we have a new release!

https://conda-forge.org/docs/maintainer/infrastructure.html#using-azure-for-everything

Reply:
You are right. Some things may fail. We can re-enable AppVeyor if that is the case.